tests/jenkins/jobs/CreateGithubIssue_Jenkinsfile.txt

```diff
@@ -0,0 +1,15 @@
+   CreateGithubIssue_Jenkinsfile.run()
+      CreateGithubIssue_Jenkinsfile.pipeline(groovy.lang.Closure)
+         CreateGithubIssue_Jenkinsfile.echo(Executing on agent [label:none])
+         CreateGithubIssue_Jenkinsfile.stage(notify, groovy.lang.Closure)
+            CreateGithubIssue_Jenkinsfile.script(groovy.lang.Closure)
+               CreateGithubIssue_Jenkinsfile.createGithubIssue({message=[Error building OpenSearch, retry with: ./build.sh manifests/2.2.0/opensearch-2.2.0.yml --component OpenSearch --snapshot]})
+                  createGithubIssue.readYaml({file=manifests/2.0.0/opensearch-2.0.0.yml})
+                  createGithubIssue.usernamePassword({credentialsId=jenkins-github-bot-token, passwordVariable=GITHUB_TOKEN, usernameVariable=GITHUB_USER})
+                  createGithubIssue.withCredentials([[GITHUB_USER, GITHUB_TOKEN]], groovy.lang.Closure)
+                     createGithubIssue.sh({script=gh issue list --repo https://github.com/opensearch-project/OpenSearch.git -S "[AUTOCUT] OS Distribution Build Failed for OpenSearch-2.0.0 in:title" --label autocut, returnStdout=true})
+                     createGithubIssue.sh({script=gh label list --repo https://github.com/opensearch-project/OpenSearch.git -S v2.0.0, returnStdout=true})
+                     createGithubIssue.sh({script=gh issue create --title "[AUTOCUT] OS Distribution Build Failed for OpenSearch-2.0.0" --body "***Recived Error***: **Error building OpenSearch, retry with: ./build.sh manifests/2.2.0/opensearch-2.2.0.yml --component OpenSearch --snapshot**.
+                      The distribution build for OpenSearch has failed.
+                      Please see build log at www.example.com/jobs/test/123" --label "autocut,v2.0.0" --repo https://github.com/opensearch-project/OpenSearch.git, returnStdout=true})
```

Comment:
nit: I think we should add `/consoleFull` too. Not everyone know where to look
Example: https://build.ci.opensearch.org/job/distribution-build-opensearch/5909/consoleFull

Reply:
makes sense. Done.